### ISSUE TITLE
Getting stuck on the implementation

### ISSUE THREAD

**Issue by ReisoAphi** [SoftwareBrothers/adminjs-import-export#15]
Im Getting this msg from the adminJs app: "You have to implement action component for your ActionSee:[the documentation](https://docs.adminjs.co/BaseAction.html)"
![image](https://user-images.githubusercontent.com/87125229/210392703-8df611ac-61d4-4567-88a9-78fc063b6468.png)

Any clue?

**Comment by joncottrell** [comment]
Same issue here. @ReisoAphi what did you end up doing?

**Comment by Brdnn** [comment]
I am facing the same issue, any workaround there?

**Comment by ZeroCho** [comment]
```
    componentLoader,
    resources: [createUserResource(componentLoader)],
```
I think you guys didn't put `componentLoader` option above `resources` option

**Comment by vikas21222** [comment]
> Im Getting this msg from the adminJs app: "You have to implement action component for your ActionSee:[the documentation](https://docs.adminjs.co/BaseAction.html)" ![image](https://user-images.githubusercontent.com/87125229/210392703-8df611ac-61d4-4567-88a9-78fc063b6468.png)
> 
>  also facing the same issue

**Comment by ahmetson** [comment]
It happens because the ComponentLoader passed to the ImportExport feature is not known by Adminjs.
You should pass the component loader that's used by Adminjs to the ImportExportFeature.

Correct format of the app.js is:
```typescript
import { ComponentLoader } from 'adminjs'

const componentLoader = new ComponentLoader()

 const adminOptions: AdminJSOptions = {
    componentLoader,
    resources: [createCustomResource(componentLoader)]
    }
 }

const adminjs = new AdminJS(adminOptions);
```

Now, the script that defines the resource with the ImportExportFeature:

```typescript
let createCustomResource = (componentLoader) => {
   return {
      resource: DatabaseModel,
      features: [
            importExportFeature({ componentLoader }),
       ],
   }
}
```

If you define the componentLoader in the script with the resource using `new ComponentLoader()` and it's not passed to the adminjs, then you will see the error.

**In short, you should have only one instane of ComponentLoader within your app.**

**Comment by lpbonomi** [comment]
@ahmetson looks like this works, maybe this should go in the docs?

**Comment by vikas21222** [comment]
> It happens because the ComponentLoader passed to the ImportExport feature is not known by Adminjs. You should pass the component loader that's used by Adminjs to the ImportExportFeature.
> 
> Correct format of the app.js is:
> 
> ```ts
> import { ComponentLoader } from 'adminjs'
> 
> const componentLoader = new ComponentLoader()
> 
>  const adminOptions: AdminJSOptions = {
>     componentLoader,
>     resources: [createCustomResource(componentLoader)]
>     }
>  }
> 
> const adminjs = new AdminJS(adminOptions);
> ```
> 
> Now, the script that defines the resource with the ImportExportFeature:
> 
> ```ts
> let createCustomResource = (componentLoader) => {
>    return {
>       resource: DatabaseModel,
>       features: [
>             importExportFeature({ componentLoader }),
>        ],
>    }
> }
> ```
> 
> If you define the componentLoader in the script with the resource using `new ComponentLoader()` and it's not passed to the adminjs, then you will see the error.
> 
> **In short, you should have only one instane of ComponentLoader within your app.**

not usefull for me still getting same error

**Comment by lpbonomi** [comment]
In my case, I wasn't bundling the components after using admin.watch() and admin.initialize() it worked

**Comment by juanjofrelopez** [comment]
@ReisoAphi Hi! Any solution to this? I'm facing the same problem with the export functionality. Thanks

**Comment by ReisoAphi** [comment]
Whenever you add a React implementation, you need to bunddle. Add to your admin app a watcher, admin.watch(), let the app run and it will bundle whenever the app sees a react change.

And dont forget to add the feature to your table features property.

**Comment by vovantics** [comment]
I too got this "You have to implement action component for your ActionSee:the documentation" error. 

I also observed the following stdout when starting my Admin.js app:
```
admin-1  | AdminJS available at http://0.0.0.0:3000/
admin-1  | "date-fns/format" is imported by "../../node_modules/@adminjs/import-export/lib/components/ExportComponent.jsx", but could not be resolved – treating it as an external dependency.
admin-1  | - Bundling files...
admin-1  | "date-fns/format" is imported by "../../node_modules/@adminjs/import-export/lib/components/ExportComponent.jsx", but could not be resolved – treating it as an external dependency.
admin-1  | No name was provided for external module "date-fns/format" in "output.globals" – guessing "format".
admin-1  | ✔ Finish bundling: 27 files
```

The problem is that ['date-fns/format' is imported in ExportComponent.tsx](https://github.com/SoftwareBrothers/adminjs-import-export/blob/v3.0.0/src/components/ExportComponent.tsx) but [defined as a dev dependency in @adminjs/import-export](https://github.com/SoftwareBrothers/adminjs-import-export/blob/v3.0.0/package.json#L38).

I worked around it by adding `"date-fns": "^4.1.0"` as a dependency in my Admin.js app.

I think the correct solution is to define "date-fns/format" as a dependency instead of a dev dependency.
